Vagrantfile

```diff
@@ -20,14 +21,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  config.ssh.insert_key = false # for vagrant 1.8.5 see https://github.com/mitchellh/vagrant/issues/7610#issuecomment-234609660

```

Comment:
You mean /opt/vagrant/embedded/gems/gems/vagrant-1.8.5/plugins/guests/linux/cap/public_key.rb ? How to execute it?